test/logger_test.js

```diff
@@ -0,0 +1,26 @@
+var test = require('tape')
+var choo = require('choo')
+var devtools = require('../')
+
+test('Filtering out logs', function (t) {
+  t.plan(1)
+
+  var app = choo()
+  app.use(devtools({
+    filter: function (eventName, data, timing) {
```

Comment:
data was always undefined in Node.js for some reason but works fine in the browser